Configs/.config/hypr/keybindings.conf

```diff
@@ -79,6 +79,7 @@ bind = $mainMod+Shift, A, exec, pkill -x rofi || $scrPath/rofiselect.sh # launch
 bind = $mainMod+Shift, W, exec, pkill -x rofi || $scrPath/swwwallselect.sh # launch wallpaper select menu
 bind = $mainMod, V, exec, pkill -x rofi || $scrPath/cliphist.sh c # launch clipboard
 bind = $mainMod, K, exec, $scrPath/keyboardswitch.sh # switch keyboard layout
+bind = $mainMod, H, exec, pkill -x rofi || $scrPath/keybinds_hint.sh c # launch keybinds hint
```

Comment:
How about changing H into slash? 

`
bind = $mainMod, slash, exec, pkill -x rofi || $scrPath/keybinds_hint.sh c # launch keybinds hint
`

* slash on keyboard had the same position with ` question ` "?"  which  which might be more intuitive. 
* ` H `  as per vim keybindings will conflict on this  too
* mostly linux uses `/` to search/query/help

Reply:
these keys aren't working when i try to bind . or / or ; do you know why? i searched everywhere for this i also wanted to bind ; with my emojis but it wasn't working as well

Reply:
Done